### ISSUE TITLE
Why not updating batch norm parameters?

### ISSUE THREAD

**Issue by emrahyigit** [kakaobrain/fast-autoaugment#50]
According to the following lines, your code doesn't update batch norm parameters. What is the real reason for that?
 
`params_without_bn = [params for name, params in model.named_parameters() if not ('_bn' in name or '.bn' in name)]`

**Comment by JiyueWang** [comment]
This line means BN layer is excluded for weight decay, not the whole BP. However, I tried the regular way and got similar results on CIFAR100. I think the link below will explain part of it. Looking forward to the official explanation.
https://blog.janestreet.com/l2-regularization-and-batch-norm/
